exercises/java/src/main/java/com/adaptionsoft/games/trivia/runner/GameRunner.java

```diff
@@ -12,20 +13,20 @@ public class GameRunner {
 	public static void main(String[] args) {
 		Game aGame = new Game();
 		
-		aGame.add("Chet");
-		aGame.add("Pat");
-		aGame.add("Sue");
+		aGame.addPlayerToGame(new Player("Chet"));
+		aGame.addPlayerToGame(new Player("Pat"));
+		aGame.addPlayerToGame(new Player("Sue"));
 		
-		Random rand = new Random(Integer.parseInt(args[0]));
+		Random randomNumber = new Random(Integer.parseInt(args[0]));
 	
 		do {
 			
-			aGame.roll(rand.nextInt(5) + 1);
+			aGame.roll(randomNumber.nextInt(5) + 1);
 			
-			if (rand.nextInt(9) == 7) {
+			if (randomNumber.nextInt(9) == 7) {
 				notAWinner = aGame.wrongAnswer();
 			} else {
-				notAWinner = aGame.wasCorrectlyAnswered();
+				notAWinner = aGame.checkAnswer();

```

Comment:
notAWinner is not the best name for what that variable is actually holding.

Reply:
I agree, I have renamed it to `isGameStillPlaying`.